packages/draw/src/mindgraph.ts

```diff
@@ -0,0 +1,73 @@
+import { GraphData } from '@mindgraph/types';
+import { Artist } from './artist';
+import { Drawable } from './canvas';
+import { RenderableLink, RenderableNode } from './renderables';
+import { Simulation } from './simulation';
+import { GraphSimulationConfig, SimulationNode } from './types';
+import { Styles } from './style';
+
+export type NodeClickCallback = (node: SimulationNode) => void;
+
+export class MindGraph {
```

Comment:
Let's update `src/index.ts` to export this at the highest level and not Artist. Also, can we update the documentation in the readme to match

Reply:
done! https://github.com/jollyjerr/mindgraph/pull/12/commits/ae109d5e021c346ede25ee924afbbbe3f1b15159 and https://github.com/jollyjerr/mindgraph/pull/12/commits/c73d90825ca405b196f8fba2b3e2f9d05226c7cc